org-ql-completing-read.el

```diff
@@ -137,10 +137,15 @@ single predicate)."
                  (marker) (if-let (it (org-entry-get marker "TODO"))
                               (concat (propertize it 'face (org-get-todo-face it)) " ")
                             ""))
+                (commit-marker (candidate)
+                  (let ((marker (gethash candidate table)))
+                    (add-text-properties 0 (length candidate)
+                     (list 'org-marker marker)
+                     candidate)))
```

Comment:
Rather than do this in a separate step and function, we should probably add the marker to a property in the `action` function when the candidate is collected.

Reply:
Moved propertization to the action. Also changed the hashtable values to just `t` since it's basically just used as a set now.

Reply:
Nevermind changing the hashtable, that messes up actually going to an entry.